elliottlib/cli/__main__.py

```diff
@@ -200,12 +208,37 @@ def find_bugs(runtime, advisory, default_advisory_type, mode, status, id, from_d
                                                                                     bug.summary[:60]))
 
     if len(flag) > 0:
-        for bug in bug_ids:
+        for bug in bugs:
             for f in flag:
                 bug.updateflags({f: "+"})
 
-    if advisory is not False:
-        elliottlib.errata.add_bugs_with_retry(advisory, [bug.id for bug in bug_ids], False)
+    if advisory is not False and not default_advisory_type:  # `--add ADVISORY_NUMBER` should respect the user's wish and attach all available bugs to whatever advisory is specified.
```

Comment:
specifying both of these is already rejected above... i think it adds only confusion to assert the same here, and would be clearer left as
```suggestion
    if advisory:  # `--add ADVISORY_NUMBER` should respect the user's wish and attach all available bugs to whatever advisory is specified.
```

Reply:
`advisory` could be either set by user (with `--add`) or `--use-default-advisory` (https://github.com/openshift/elliott/pull/142/files#diff-13669dcc79bc03ff33da296c047bd280R182).

We will attach all bugs if that variable is set by `--add` not `--use-default-advisory`.

Reply:
i see; alright then!